### ISSUE TITLE
no-unexpected-multiline should warn for tagged template strings [$15]

### ISSUE THREAD

**Issue by feross** [eslint/eslint#4210]
`no-unexpected-multiline` should warn for this case:

```js
let x = function() {}
`hello`
```

To prevent it from being parsed like this:

```js
let x = function() {}`hello`
```

For the full previous discussion about this topic, see: https://github.com/eslint/eslint/issues/4169#issuecomment-149064446

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.
